gulpfile.js

```diff
@@ -13,6 +13,10 @@ gulp.task('lint', function() {
     .pipe(eslint.format());
 });
 
+gulp.task('dev', function() {
+  return gulp.watch('src/**/*', ['src']);
```

Comment:
Yes. Locally I have element-zones linked into my polydev client, and quite often I'm making changes to element-zones.